src/frontend/src/lib/components/tokens/TokenCard.svelte

```diff
@@ -17,7 +19,12 @@
 		aria-label={`Open the list of ${token.symbol} transactions`}
 	>
 		<Card noMargin>
-			{token.name}
+			{#if isTokenCkLedger(token)}
+				<Tag>ck</Tag>
```

Comment:
A screen reader will read this as "ckErc20" or will read it as "ck erc20"?

Reply:
for now "ck erc20"... there is a whitespace since it is inside a `<span>` but we can remove it. I have no strong opinion on this.

What  is better?

For example "USD Coin": what is the difference in having either "ckUSD Coin" or "ck USD Coin"? Or, with USDT: "ckTether USD" or "ck Tether USD"? What is preferable?

Reply:
I assume that showing "ck USD" is not accurate in term of accessibility.

Reply:
included in last commit, have a look please